### PR TITLE
doc: how to join us as a contributor

### PR DESCRIPTION
**Pull request by dricazenck**
* Add on README how to contribute session:

> image

* Changed how to contribute to team's page

> image

* Also `Contributing Guidelines` based in [`wwcode - london`](https://github.com/WomenWhoCode/WWCode-London/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by yuqingwang98** [comment]
one small suggestion: the text on the button could be 'click here to learn more'

File: team.html [hunk 1]
```diff
@@ -77,7 +77,7 @@ <h5 class="card-title">{{person.name}}</h5>
         <h1>Feel free to join us as a contributor!</h1>
 
         <p class="mt-4 d-flex justify-content-center">
-            <a href="https://wwcodelondon.slack.com/archives/CJ0S3FZK8" target="_blank" class="btn btn-primary">Join our slack channel</a>
+            <a href="https://github.com/WomenWhoCode/london#contributing" target="_blank" class="btn btn-primary">Check here how</a>
```
Comment (dricazenck) [inline team.html:80]:
```suggestion
            <a href="https://github.com/WomenWhoCode/london#contributing" target="_blank" class="btn btn-primary">click here to learn more</a>
```

**Review by nora-weisser — APPROVED** [review]

**Comment by goldfishdolphin** [comment]
> one small suggestion: the text on the button could be 'click here to learn more'

'click here to learn more' sounds better.

**Comment by dricazenck** [comment]
> 'click here to learn more' sounds better.
Yes, I already fix that. thanks

**Pull request merged**